pkg/webhook/server/server.go

```diff
@@ -59,7 +58,13 @@ func WithCertDir(certDir string) ServerModifier {
 	}
 }
 
-func (s *server) Start(stop <-chan struct{}) error {
+func (s *Server) UpdateOpts(serverOpts []ServerModifier) {
```

Comment:
Can this be called after the server is started? Please document it.

Also, usually we'd use a single argument of variable length https://gobyexample.com/variadic-functions, that should be easier to use than initializing a list just to pass arguments

Reply:
I believe once the manager is started these parameters cannot be updated (@qinqon - am I right?).